package.json

```diff
@@ -31,17 +31,18 @@
       "scripts/create-component/package-template/**/*"
     ],
     "linters": {
+      "**/*.css": [
+        "prettier-eslint --write",
+        "git add"
+      ],
       "**/*.js": [
         "prettier-eslint --write",
+        "eslint",
         "git add"
       ],
       "package.json": [
         "format-package -w",
         "git add"
-      ],
-      "packages/**/*.css": [
-        "node_modules/.bin/prettier --single-quote --no-semi --write",
```

Comment:
Without the single quote and no semi flags, are those getting read from somewhere else that's committed to the project?  We should get a .prettierrc in here, but we don't have one yet.

Reply:
I am blind. :)